src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -488,9 +488,13 @@ private function getType(string $type, bool $isCollection, string $className = n
      */
     private function computeDoc(Documentation $documentation, \ArrayObject $definitions, \ArrayObject $paths) : array
     {
+        /** @var Symfony\Component\Routing\RequestContext $requestContext */
+        $requestContext = $this->urlGenerator->getContext();
```

Comment:
Would [configuring the request context](http://symfony.com/doc/current/console/request_context.html#configuring-the-request-context-globally) not work here as well? We fetch host and scheme here, if they return null, we throw (maybe with a hint what to do).